nc/include/component/PhysicsBody.h

```diff
@@ -4,34 +4,56 @@
 
 namespace nc
 {
+    /** Notes on physics bodies
+     *  - A PhysicsBody must be added to an Entity with a Collider.
+     *  - Static Entities will automatically have a mass of 0.
+     *  - Translations and rotations can be resticted on a per-axis
+     *    basis with linear/angularFreedom.
+     */
     struct PhysicsProperties
     {
-        Vector3 velocity = Vector3::Zero();        // Current linear velocity
-        Vector3 angularVelocity = Vector3::Zero(); // Current angular velocity
-        Vector3 linearFreedom = Vector3::One();    // Per-axis translation restriction (use 0 to lock)
-        Vector3 angularFreedom = Vector3::One();   // Per-axis rotation restriction (use 0 to lock)
-        float mass = 1.0f;                         // Mass in kilograms (0 is infinite)
-        float drag = 0.05f;                        // Linear resistence
-        float angularDrag = 0.1f;                  // Angular resistence
-        float restitution = 0.3f;                  // Coefficient of restitution (collision elasticity) [0, 1]
-        float friction = 0.5f;                     // Coefficient of friction [0, 1]
-        float baumgarte = 0.4f;                    // Position correction factor to apply on collision [0, 1]
-        bool useGravity = true;                    // Is the object affected by gravity
+        float mass = 1.0f;        // Mass in kilograms (0 is infinite)
+        float drag = 0.05f;       // Linear resistence
+        float angularDrag = 0.1f; // Angular resistence
+        float restitution = 0.3f; // Coefficient of restitution (collision elasticity) [0, 1]
+        float friction = 0.5f;    // Coefficient of friction [0, 1]
+        float baumgarte = 0.4f;   // Position correction factor to apply on collision [0, 1]
+        bool useGravity = true;   // Is the object affected by gravity
+        bool kinematic  = false;  // Prevent forces and collisions from affecting the object
```

Comment:
will you change to isKinematic? I prefer booleans in question form  to easily identify them from other properties